jekyll/_cci2/first-steps.adoc

```diff
@@ -0,0 +1,80 @@
+---
+contentTags: 
+  platform:
+  - Cloud
+---
+= Sign up and try CircleCI
+:page-layout: classic-docs
+:page-liquid:
+:page-description: First step for using CircleCI. 
+:icons: font
+:toc: macro
+:toc-title:
+
+To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your GitHub or Bitbucket account, or your email address for the option to connect to your code (including your GitLab projects) later in the process.
```

Comment:
It could just be me, but this sentence sounds a little off. I don't think we should put GitLab instructions in parentheses in the intro. It's also a little confusing because we now have a button that says GitLab on the main Sign Up site that also just directs a user to the same page as the email method, but there is _also_ a button for email. Maybe just split this into something that directly references GitLab more...

> Sign up with your GitHub or Bitbucket account directly, or your email address for the option to connect to your code later in the process. If you would like to connect code from GitLab, you will go through the email sign-up process rather than use your GitLab account directly.

Reply:
Thanks, good catch! I've just put GL on an equal footing with GH and BB.